### PR TITLE
Pi logging #21

### PR DESCRIPTION
**Pull request by fergusL**
PR to address #21

Have replaced the `logging` module with the `logbook` module and implemented loggers for the python grpc server and the `domehunter.dome_control.Dome` class. Both the server and the Dome class will log to terminal and a logfile. The log level of each (server/dome and the file/terminal logger) can be set to different log levels. Although I think the log level of the file handler has to be lower than the terminal handler, as it is lower in the handler hierarchy...(?)

I also snuck in a small change that allows the LED debugging light brightness to be set via keyword arg or in the yaml config. Also have added an ASCII art huntsman logo that can be displayed upon initialising the dome sever/Dome class.

Starting this as a PR draft as I haven't tested it on the actually observatory. Will do that before merging.

Another thing to mention is that this doesn't address #20, which is the logging done within the TheSkyX gRPC plugin. At the moment these logs can be view from the TheSkyX logging window but I haven't looked into how to get the logs into a logfile (is this needed?).

### CODE REVIEW

File: domehunter/gRPC-server/dome_controller_config.yml [hunk 1]
```diff
@@ -1,11 +1,17 @@
 ---
 name: Huntsman Dome Controller
+logo: True
+log_file_level: 'DEBUG'
```
Comment (lspitler) [inline domehunter/gRPC-server/dome_controller_config.yml:13]: Is it useful to provide other logger level options here? Commented out?

**Review by lspitler — APPROVED** [review]
LGTM. One minor comment on the code it self.

The main thing I'd like to see is a change on a few of the logger outputs: three log events are fun cultural references, which is fine, except when someone doesn't understand them. For I don't remember where this comes from "ITS DOING IT, ITS DOING IT". I had to look at the function to get what it means, which kind of defeats the purpose of log info. So how about add a concise statement to the fun cultural reference? So:  `"ITS DOING IT, ITS DOING IT - Go To AzEl successfully completed"`

**Comment by lspitler** [comment]
aww, you removed the fun bits @fergusL . :-) note this is still tagged WIP in github, so no formal review can take place...

**Comment by fergusL** [comment]
@lspitler yea they weren't really cultural references (except phoning home I guess) just random stuff I wrote in early on when I was bored. I'm just leaving it as a draft PR until I've tried it out with the actual dome hardware and made sure none of the changes cause problems

**Comment by fergusL** [comment]
Have tested changes within this pr on the actual dome and nothing appears to have broken. Will also be updating the huntsman-pocs wiki with instructions for setting up the dome server etc

**Comment by lspitler** [comment]
OK @fergusL  let us know if you'd like further review or in particular from someone else.

**Pull request merged**